instagram/src/components/AppContainer/PostContainer/PostComponents/CommentContainer/CommentContainer.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import IconsLikes from "./IconsLikes";
+import Comments from "./Comments/Comments";
+import AddComment from "./AddComment";
+
+const CommentSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`;
+
+class CommentContainer extends Component {
+  state = {
+    comments: [],
+    comment: "",
+    likes: 0,
+    liked: false,
+    showComment: false
+  };
+
+  componentDidMount() {
+    this.setState(() => {
+      return { comments: [...this.props.comments], likes: this.props.likes };
+    });
+  }
+
+  likePost = () => {
```

Comment:
This gets the job done but it's a bit verbose. You could've done: 

likePost = () => {
    let numOfLikes = this.state.likes;

    this.state.liked === false ? numOfLikes += 1 : numOfLikes -= 1;

    this.setState({
      likes: numOfLikes,
      liked: !this.state.liked
    })
  }